about.html

```diff
@@ -0,0 +1,142 @@
+    <nav class="nav-class">
+      <div class="ham_menu_btn">
+        <a href="" id="ham_menu_button">
+          <img src="imgs/mobile_menu_btn.png" alt="hamburger menu">
+        </a>
+      </div>
+      <ul class="mobile-nav">
+        <li class="mobile-item"><a href="#" id="close_mobile_menu"><img id="nav-close-btn" src="imgs/close_mobile_menu.png" alt="close button"></a>
+        </li>
+        <li class="mobile-item"><a href="index.html">Home</a></li>
+        <li class="mobile-item"><a href="about.html">About</a></li>
+        <li class="mobile-item">Join</li>
+        <li class="mobile-item" id="pres"><a href="#">Presentation</a></li>
+      </ul>
+      <ul class="desktop_menu">
+        <li class="menu_item intro"><a href="about.html">About</a></li>
+        <li class="menu_item program">Program</li>
+        <li class="menu_item application">Join</li>
+        <li class="menu_item sponsor">Sponsor</li>
+        <li class="menu_item news">News</li>
+        <li class="menu_item presentation">
+          <a href="#">CC Campaign</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="mobile-logo-wrap">
+      <a href="index.html">
+        <img src="imgs/logo.png" alt="CC Global Summit 2015">
+      </a>
+    </div>
+  </header>
+  <main>
+    <section id="intro-sectn">
+      <header class="intro-sctn-header">
+        <h1 class="intro-sctn-title">
+          <span id="intro-sctn-span">"Hello, Sharing World!"</span>
+          AUGMENTED REALITY ART SUMMIT 2022
+        </h1>
+      </header>
+      <div class="intro-msg">
+        <p>As a year of technological shifts, 2022 changed the art and metaverse industry with new trends, synergies, and key players. Creatives increasingly understood the importance of collaboration, going beyond the obvious and using new mediums in their work, such as AR, VR, NFTs, etc. The AR Art Summit was born as a reflection of the current shifts happening in the art world.
+        <p id="second-para">Creatives increasingly understood the importance of collaboration, going beyond the obvious and using new mediums in their work, such as AR, VR, NFTs, etc. The AR Art Summit was born as a reflection of the current shifts happening in the art world.
+        </p>
+        </p>
+      </div>
+      <div class="contact-info">
+        <div class="info-msg">
+          <p>Please contact us per Email for any further questions about AR Art Summit 2022!</p>
+        </div>
+        <div class="email-info">
+          <p>arartsummit2022@arkorea.org</p>
+        </div>
+      </div>
+    </section>
+    <section id="logo-sctn">
+      <header class="logo-headr">
+        <h2>AR Art Summit 2022 Logo</h2>
+        <div class="orange-bar"></div>
+      </header>
+      <div class="logo-info">
+        <p>The logo of AR Art Summit 2022 was decided through the logo competition from 8 June to 7 July.</p>
+      </div>
+      <div id="logo-image">
+        <img src="imgs/ar-art-summit-2022-600x315.jpeg" alt="summit logo">
```

Comment:
- [ ] kindly reduce the size of the image that represent summit logo to match the template design


![image](https://user-images.githubusercontent.com/89282221/190816303-ddd4dd13-a803-47e6-89cf-9b6ea368e79e.png)